styles.css

```diff
@@ -32,8 +32,10 @@
   }
 
   button,
```

Comment:
- Good Job building the form for the contact form. However, according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.), you are supposed stick to the design as much as possible  using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

**Figma Design**
![Screenshot from 2023-05-02 14-18-08](https://user-images.githubusercontent.com/97834160/235667028-627136e8-9775-4821-90ff-9ca589a824df.png)

**Your Design**
![Screenshot from 2023-05-02 14-18-24](https://user-images.githubusercontent.com/97834160/235667050-c3972c17-a15b-4bba-b16a-f4d5b145a0d5.png)


- Kindly make sure to stick to the colors, font-weight and sizes for the text, the button and the icons.